src/node_crypto.cc

```diff
@@ -5512,7 +5522,18 @@ void RandomBytesCheck(RandomBytesRequest* req, Local<Value> argv[2]) {
     size_t size;
     req->return_memory(&data, &size);
     argv[0] = Null(req->env()->isolate());
-    argv[1] = Buffer::New(req->env(), data, size).ToLocalChecked();
+    Local<Value> buffer = req->object()->Get(req->env()->buffer_string());
+
+    if (Buffer::HasInstance(buffer)) {
+      char* buf = reinterpret_cast<char*>(Buffer::Data(buffer));
+      const size_t max = req->offset() + size;
+      for (size_t i = req->offset(); i < max; i++) {
+        buf[i] = data[i - req->offset()];
+      }
```

Comment:
If you want to keep the separate copy, you can probably replace this by a call to `memcpy()`